packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -593,22 +611,22 @@ class VirtualListBasic extends Component {
 
 	calculateMetrics (props) {
 		const
-			{clientSize, direction, itemSize, overhang, spacing} = props,
-			node = this.props.scrollContentRef.current;
+			{direction, itemSize, overhang, spacing} = props,
+			contentSizes = this.getContentSize(props);
```

Comment:
Why it's Sizes?

Reply:
I named it since it has two values: clientHeight and clientWidth. Just 'size' would be good to me also.